src/Containers/Project/Project.js

```diff
@@ -1,27 +1,27 @@
 import React, { Component } from 'react';
-import MiniPaletteArea from '../MiniPaletteArea/MiniPaletteArea';
+import MiniPaletteArea from '../../Components/MiniPaletteArea/MiniPaletteArea';
 import { fetchProjectPalettes } from '../../Thunks/fetchProjectPalettes';
 import { setCurrentExpandedProject } from '../../Actions';
 import { connect } from 'react-redux';
 
 export class Project extends Component{
-  constructor() {
-    super();
-    this.state = {
-      expanded: false
+  fetchPalettes = async () => {
+    const { id } = this.props
+    if(!this.props.palettes.length){
```

Comment:
Nice work! Glad we're only fetching palettes if the project doesn't have any palettes in redux